.github/config.yml

```
@@ -0,0 +1,21 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thank you for opening your first issue here! :tada: Be sure to follow the issue template :)
```

```suggestion
  Thank you for opening your first issue here! :tada:
```

> Be sure to follow the issue template :)

Not sure this part is needed, since the issue could only be opened via the issue template due to https://github.com/WorldHealthOrganization/open-source-communication-channel/blob/d50618dca809c51a78d4dfa71fd78345df930de3/.github/ISSUE_TEMPLATE/config.yml#L1